hopsworks-api/src/main/java/io/hops/hopsworks/api/admin/SystemAdminService.java

```diff
@@ -91,4 +99,54 @@ public Response changeMasterEncryptionPassword(@Context SecurityContext sc, @Con
           "password file: " + ex.getMessage());
     }
   }
+  
+  @PUT
+  @Path("/variables/refresh")
```

Comment:
@kouzant @augustbonds same as previous comment, "refresh" is implied by PUT. Maybe this one can be POST and the next one can remain PUT. So we can differentiate between the two.

Reply:
I set both PUT as neither produces content.

Reply:
Actually, the important thing about PUT is that it is idempotent. That means, that the state on the server should not change again when requesting twice. I think this is actually a POST.